app/routes/docs.client.samples.md

```diff
@@ -164,3 +166,62 @@ for message in consumer.consume(end[0].offset - start[0].offset, timeout=1):
             continue
     print(message.value())
 ```
+
+# Working With JSON Schema
+
+GCN Notices are distributed in JSON format, this guide describes how to handle JSON schema.
+
+## Parsing JSON Notices
+
+Read the JSON data from [sample.schema.json](https://gcn.nasa.gov/docs/notices/schema) and [sample.example.json](https://gcn.nasa.gov/docs/notices/schema), parses it into Python dictionaries.
+
+```python
+import json
+
+# Load and parse schema and example JSON files
+with open('sample.schema.json', 'r') as schema_file:
+    schema = json.load(schema_file)
+
+with open('sample.example.json', 'r') as example_file:
+    example = json.load(example_file)
+
+print('Schema:', schema)
+print('Example:', example)
+```
+
+## Encoding and Decoding of Embedded Data
+
+The following code sample shows how to encode/decode a file in Python. The `base64` package includes the methods `b64decode` and `b64encode` to make this task simple.
```

Comment:
```suggestion
The following code shows how to encode/decode a GCN Notice using the `b64decode` and `b64encode` methods from the `base64` Python package.
```

Reply:
I still find the previous more descriptive.

Reply:
Provide hyperlinks to these Python methods.